src/wallet/coincontrol.h

```diff
@@ -53,6 +53,8 @@ class CCoinControl
     bool m_avoid_partial_spends = DEFAULT_AVOIDPARTIALSPENDS;
     //! Forbids inclusion of dirty (previously used) addresses
     bool m_avoid_address_reuse = false;
+    //! Prioritize a solution that doesn't generate change when selecting coins
+    bool m_prioritize_no_change = false;
```

Comment:
Should this be in `CoinSelectionParams` instead?

Reply:
It's in both I guess